### ISSUE TITLE
Inconsistent button/link guidance on the "Navigate a long list" pattern

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/vets-design-system-documentation#1613]
### Duplicate check

- [X] I've searched for any related issues and avoided creating a duplicate issue.

### This update is for:

Pattern

### What is the name?

Navigate a long list

### What is the nature of this update?

- [ ] How to build this component/pattern
- [ ] When to use this component/pattern
- [ ] When to use something else
- [ ] Usability guidance
- [X] Accessibility
- [ ] Implementation
- [ ] Research insights
- [ ] Package information
- [ ] Addition to Forms Library documentation
- [ ] Update to existing Forms Library documentation

### What problem does this solve?

The two examples provided on "Navigate a long list" are inconsistent in what element triggers the list to expand.
- In the VAOS example, a link is used.
- In the Claim Status tool example, a button is used.
   - In the "Anatomy or layout details" section, the wireframes also use a button.

From an accessibility standpoint, the correct answer is button! But the "How this pattern works" section references both buttons and links as valid triggers:

> - Given a long list of options, only the first 5 options are displayed.
> - A button or link below the options acts as a trigger for revealing additional options.
>    - This button or link does not appear if there are 5 or less options.
> - Clicking the button or link displays more options, 5 at a time.
> - Once all options are displayed, the button or link disappears, or changes label (“Hide options”).
> - Once the additional options are revealed, there may be no need, and thus no mechanism, to hide them.

Could we get this clarified that the trigger should be a button (styled as a button)?

The accessibility considerations referencing ARIA attributes could also be expanded a little -- happy to help with that. I'm not sure how practical it would be to component-ize this pattern, but I'd be in favor of that too if it's possible to just provide a single standardized control.

**Comment by humancompanion-usds** [comment]
Yeah, I think the problem here @briandeconinck is that I was trying to not invalidate the variations of this pattern that use a link. So essentially grandfathering VAOS in. We could also really use a tertiary button here because the secondary button used in this pattern is just oversized for the job at hand. Did you all file an issue against VAOS to have them change their design from link to button?
